plugins/modules/slack.py

```diff
@@ -143,6 +144,38 @@
       - 'never'
       - 'auto'
     version_added: 6.1.0
+  upload_file:
+    type: dict
+    description:
+      - Specify details to upload a file to Slack. The file can include metadata such as an initial comment, alt text, snipped and title.
+      - See Slack's file upload API for details at U(https://api.slack.com/methods/files.getUploadURLExternal).
+      - See Slack's file upload API for details at U(https://api.slack.com/methods/files.completeUploadExternal).
```

Comment:
These are two paragraphs with the same text, but different URLs. Either write something like
```suggestion
      - See Slack's file upload API for details at U(https://api.slack.com/methods/files.getUploadURLExternal) or
        U(https://api.slack.com/methods/files.completeUploadExternal).
```
or adjust the texts.